etc/requirements_alpaca.txt

```diff
@@ -0,0 +1 @@
+alpaca-trade-api>=0.9
```

Comment:
I'd recommend depending on an exact version (or at version range with an upper boundary) so that we're not hit by breaking API changes in future versions.

Reply:
Alpaca Trade API in general will not break compatibility (if it does, it is really a big problem for us), and it updates frequently to improve the quality as it is still relatively new.  I hear it depends on the project policy so I can stick with a version and keep updating this file frequently, but my personal opinion is zipline still sticking to 0.18 is such a legacy.  I think the latest convention is not to use this file to lock down the version, but rather should use something like Pipfile.lock.  All other languages nowadays use some sort of lock file (ruby, go, js, etc) and python is coming to this partly late with pipenv.

Reply:
That's fine. I think it is in your best interest to keep the integration working :)